src/discussions/learners/LearnersView.test.jsx

```diff
@@ -53,41 +63,190 @@ describe('LearnersView', () => {
         roles: [],
       },
     });
-
+    axiosMock = new MockAdapter(getAuthenticatedHttpClient());
     store = initializeStore();
     Factory.resetAll();
+  });
+
+  async function setUpLearnerMockResponse(
```

Comment:
can setUpLearnerMockResponse from `frontend-app-discussions/src/discussions/learners/test-utils.js` be utilised here by tweaking it a little bit?